subz_commands_aclr8.py

```diff
@@ -64,4 +103,59 @@ def on_fail(self, path, is_temporary):
 
   def remove_temp(self, path, is_temporary):
     if is_temporary:
-      os.remove(path)
\ No newline at end of file
+      os.remove(path)
+
+class SubzAclr8QueryCoupled(sublime_plugin.WindowCommand):
```

Comment:
there's a lot of duplication with `SubzAclr8TestsCoupled`. Would it be worth it to remove it, e.g. via inheritance?